src/asset_folder_vsingester.py

```diff
@@ -7,18 +7,18 @@
 #this also requires python-setuptools to be installed
 from asset_folder_importer.config import configfile
 from optparse import OptionParser
-from Queue import Queue
+from queue import Queue
 from asset_folder_importer.asset_folder_vsingester.exceptions import *
 from asset_folder_importer.asset_folder_vsingester.importer_thread import *
+import importlib
 
 MAXTHREADS = 8
 #suid perl script so we don't need to run the whole shebang as root
 PERMISSIONSCRIPT = "/usr/bin/asset_permissions.pl"
 #set default encoding to utf-8 to prevent template errors
 XML_CHECK_TIMEOUT = 60  #wait up to 60s for XML validation
 import sys
-reload(sys)
-sys.setdefaultencoding('utf-8')
+importlib.reload(sys)
```

Comment:
not sure that this is necessary, think it was only for that `setdefaultencoding` hack that we don't need any more

Reply:
Removed.